### PR TITLE
added code and files for HELP generation

### PR DESCRIPTION
**Pull request by danduk82**
These are the files needed to generate the Help webpage.

### CODE REVIEW

File: help/index.html [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		</div>
+	<div id="footer">
+	<p>Footer Text</p>
+	</div>	
+</div>
+<script src="js_css/marked.min.js"></script>
+<script src="js_css/jquery-1.9.1.js"></script>
+<script src="js_css/script.js"></script>
+</body>
+</html>
```
Comment (cedricmoullet) [inline help/index.html]: Missing new line

**Comment by gjn** [comment]
Any reason why this isn't put into separate git repository? Hosting?

**Comment by cedricmoullet** [comment]
Keep everything in the same place.And the help is directly related with the application.
But of course, it would be possible to create a separate repository. I have no strong opinion regarding this.

**Comment by cedricmoullet** [comment]
@oterral @gjn do we go ahead or do you want it in another location ?

**Comment by oterral** [comment]
for me it's good to have this in this repository. I let @gjn decide if he wants it elsewhere.

**Pull request merged**